src/renderers/webgl/WebGLRenderLists.d.ts

```diff
@@ -23,16 +23,24 @@ export interface RenderItem {
 
 export class WebGLRenderList {
   opaque: Array<RenderItem>;
-  transparent: Array<any>;
+  transparent: Array<RenderItem>;
   init(): void;
   push(
     object: Object3D,
     geometry: Geometry | BufferGeometry,
     material: Material,
+    groupOrder: number,
+    z: number,
+    group: Group
+  ): void;
+  unshift(
+    object: Object3D,
+    geometry: Geometry | BufferGeometry,
```

Comment:
The type of the geometry is always `BufferGeometry`. This is ensures by `WebGLObjects`.

Reply:
also whoa

https://github.com/mrdoob/three.js/blob/0d18a759b5de4978f16ec51cf4efd8ae677724c2/src/renderers/WebGLRenderer.js#L1294

Reply:
OK I addressed both, thank you to point out this :pray: